packages/slate-editor/src/lib/isSupportedImageOrigin.ts

```diff
@@ -0,0 +1,6 @@
+const CORS_ENABLED_ORIGINS = ['https://cdn.uc.assets.prezly.com'];
+
+export function isSupportedImageOrigin(url: string) {
```

Comment:
Not sure if the name is okay or if it should be called something like `isCorsEnabledOrigin`? 🤔

Reply:
I think `isCorsEnabledOrigin` suits better, since the current name doesn't make clear what exactly is supported 🤔